packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -308,6 +309,10 @@ class NavigationRail extends StatefulWidget {
   /// when [useIndicator] is true.
   final Color? indicatorColor;
 
+  /// Overrides the default value of [NavigationRail]'s selection indicator shape,
```

Comment:
The indicatorColor and indicatorShape properties should explain how they're related to NavigationRail and NavigationRailTheme and what the default values are.